README.md

```diff
@@ -45,68 +50,7 @@ const ReaderPage = ({ manifestUrl }) => {
 };
 ```
 
-Passing a content decryptor to the reader for use by the Client. This would be how we render AxisNow content for example:
-
-```typescript
-import WebReader from "nypl/web-reader"
-import AxisNowDecryptor from "nypl/axisnow-access-control-web"
-
-const ReaderPage = ({manifestUrl}) => {
-  const decryptor = new AxisNowDecryptor(...);
-  return (
-	  <WebReader
-      getContent={decryptor.getContent}
-      manifestUrl={manifestUrl}
-    />
-  )
-}
-```
-
-To support customization, you can piece together your own UI and call the `useWebReader` hook to get access to the reader API.
```

Comment:
I attempted to use the `useWebReader` hook and ran into several issues, so I removed this example. The `usePDFReader` and `useHTMLReader` hooks worked fine though, and I added working examples for those in the example app and linked them in the README.

Lmk if you think I should create a ticket to fix/implement the useWebReader hook.

Reply:
Interesting. No I think it's fine to have just `useHtmlReader` and `usePdfReader` for now